src/utils/index.ts

```diff
@@ -165,3 +165,29 @@ export const getLocalizedObject = (localizations: LocalizedObject[]) => {
         return localizations[0];
     }
 };
+
+export interface ClickHandler {
+    onMouseDown: (event: React.MouseEvent<HTMLDivElement>) => void;
+    onMouseUp: (event: React.MouseEvent<HTMLDivElement>) => void;
+}
+
+/**
+ * Creates a simple click handler via onMouseDown and onMouseUp.
+ * This is a necessity in order to let jsPlumb manage our element dragging.
+ * @param onClick
+ */
+export const createClickHandler = (
+    onClick: (event: React.MouseEvent<HTMLDivElement>) => void
+): ClickHandler => {
+    return {
+        onMouseDown: (event: React.MouseEvent<HTMLDivElement>) => {
+            this._clicked = true;
```

Comment:
why not push `_clicked` to component state?

Reply:
This would require any component that uses this to have `clicked` as part of its state. Not sure that's worth it since this lives only in the instance between mouse down and mouse up.